src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
```

Comment:
Wrap if you have more than 2 - 3 params
```suggestion
  const { 
    name, 
    age, 
    sex, 
    isMarried, 
    partnerName
   } = person;
```